ts/features/idpay/initiative/details/components/timelineItems/CardTransaction.tsx

```diff
@@ -0,0 +1,109 @@
+
+type TransactionProps<T> = { transaction: T };
+
+const styles = StyleSheet.create({
+  alignCenter: {
+    alignItems: "center"
+  },
+  spaceBetween: {
+    justifyContent: "space-between"
+  }
+});
+
+const getHourAndMinuteFromDate = (date: Date) =>
+  `${String(date.getHours()).padStart(2, "0")}:${String(
+    date.getMinutes()
+  ).padStart(2, "0")}`;
+
+type BaseTransactionCardProps = { children: React.ReactNode };
+const BaseTransactionCard = ({ children }: BaseTransactionCardProps) => (
+  <ListItem style={styles.spaceBetween}>
+    <View style={[IOStyles.flex, IOStyles.row, styles.alignCenter]}>
+      {children}
+    </View>
+  </ListItem>
+);
+export const TimelineTransactionCard = ({
+  transaction
+}: TransactionProps<TransactionOperationDTO>) => (
+  <BaseTransactionCard>
+    <Text>LOGO</Text>
+    <View hspacer />
+    <View style={IOStyles.flex}>
+      <H4>{transaction.operationType}</H4>
+      <LabelSmall weight="Regular" color="bluegrey">
+        {`${formatDateAsShortFormat(
+          transaction.operationDate
+        )}, ${getHourAndMinuteFromDate(
+          transaction.operationDate
+        )} · ${formatNumberAmount(Math.abs(transaction.amount), true)}`}
+      </LabelSmall>
+    </View>
+    <H4> {`${formatNumberAmount(transaction.amount, false)} €`}</H4>
+  </BaseTransactionCard>
+);
+
+export const OnboardingTransactionCard = ({
+  transaction
+}: TransactionProps<OnboardingOperationDTO>) => (
+  <BaseTransactionCard>
+    <Text>LOGO</Text>
+    <View hspacer />
+    <View style={IOStyles.flex}>
+      <H4>{transaction.operationType}</H4>
+      <LabelSmall weight="Regular" color="bluegrey">
+        {`${formatDateAsShortFormat(
+          transaction.operationDate
+        )}, ${getHourAndMinuteFromDate(transaction.operationDate)}`}
+      </LabelSmall>
+    </View>
+  </BaseTransactionCard>
+);
+
+export const InstrumentOnboardingCard = ({
+  transaction
+}: TransactionProps<InstrumentOperationDTO>) => (
+  <BaseTransactionCard>
+    <Text>LOGO</Text>
+    <View hspacer />
+    <View style={IOStyles.flex}>
+      <H4>
+        {transaction.operationType} {transaction.maskedPan}
+      </H4>
+      <LabelSmall weight="Regular" color="bluegrey">
+        {`${formatDateAsShortFormat(
+          transaction.operationDate
+        )}, ${getHourAndMinuteFromDate(transaction.operationDate)}`}
+      </LabelSmall>
+    </View>
+  </BaseTransactionCard>
+);
+
+export const IbanOnboardingCard = ({
+  transaction
+}: TransactionProps<IbanOperationDTO>) => (
+  <BaseTransactionCard>
+    <Text>IBANLOGO</Text>
+    <View hspacer />
+    <View style={IOStyles.flex}>
+      <H4>{transaction.operationType}</H4>
+      <LabelSmall weight="Regular" color="bluegrey">
+        {`${formatDateAsShortFormat(
+          transaction.operationDate
+        )}, ${getHourAndMinuteFromDate(transaction.operationDate)}`}
+      </LabelSmall>
+    </View>
+  </BaseTransactionCard>
+);
```

Comment:
I think there is no need to create a nested folder for one component. Maybe you should consider moving `CardTransaction` one folder up and remove `timelineItems` folder.